sklearn/cluster/tests/test_k_medoids.py

```diff
@@ -0,0 +1,150 @@
+"""Testing for K-Medoids"""
+import numpy as np
+from scipy.sparse import csc_matrix
+
+from sklearn.cluster import KMedoids, KMeans
+from sklearn.datasets import load_iris
+from sklearn.metrics.pairwise import PAIRWISE_DISTANCE_FUNCTIONS
+from sklearn.metrics.pairwise import euclidean_distances
+from sklearn.utils.testing import assert_array_equal
+from sklearn.utils.testing import assert_equal, assert_raise_message
+from sklearn.utils.testing import assert_greater
+from sklearn.utils.testing import assert_allclose
+
+rng = np.random.RandomState(0)
```

Comment:
making this global means that the addition of a test, or a pairwise distance metric, could make an arbitrary test fail